### PR TITLE
Fix centos 1/5

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: rpm_spec/qubes-utils.spec [hunk 1]
```diff
@@ -35,26 +39,38 @@ Common Linux files for Qubes Dom0 and VM
 Summary:    Python package qubesimgconverter
 Requires:   python
 Requires:   pycairo
+%if 0%{rhel} >= 7
```
Comment (marmarek) [inline rpm_spec/qubes-utils.spec]: Missing `?`, spec parsing fails - see travis log

Comment (fepitre) [reply rpm_spec/qubes-utils.spec]: I'm correcting. Hopefully and helpfully travis is here...

**Review by fepitre — COMMENTED** [review]
I did the change (sorry I prefer to force push instead of having lot of commits)

**Comment by fepitre** [comment]
Do you have a quick way to add to .travis.yml the CentOS template? It would be helpful to debug and track problems related to CentOS. I still have not finished my integration platform with Jenkins...

**Comment by marmarek** [comment]
Should be easy - add appropriate line to .travis.yml here (DISTS_VM=centos7 ..., see existing ones). builder-centos plugin is already enabled in [travis-builder.conf](https://github.com/QubesOS/qubes-builder/blob/master/scripts/travis-builder.conf)

**Comment by marmarek** [comment]
You can even do it in this PR and see :)

**Comment by marmarek** [comment]
As for the whole template - see .travis.yml in builder-fedora, you can do similar thing in builder-centos

**Comment by fepitre** [comment]
Sure. I meant a quick way for all the packages... If not it looks like it will be one PR for each..

**Comment by marmarek** [comment]
Unfortunately you're correct...

**Comment by fepitre** [comment]
Ok. I'm on it :)

**Review by marmarek — APPROVED** [review]

**Pull request merged**